SCClassLibrary/Common/Collections/Scale.sc

```diff
@@ -19,8 +19,12 @@ Scale {
 	}
 
 	*doesNotUnderstand { |selector, args|
-		var scale = this.newFromKey(selector, args).deepCopy;
-		^scale ?? { super.doesNotUnderstand(selector, args) };
+		^this.newFromKey(selector, args).deepCopy
```

Comment:
ok, this is cosmetic, but since you changed it, here a suggestion for readability:

```supercollider
    var scale = this.newFromKey(selector, args);
    if(scale.notNil) { ^scale.deepCopy };
    ^super.doesNotUnderstand(selector, args)
```